examples/compose/plot_pipeline_display.py

```diff
@@ -19,16 +19,15 @@
 
 Note that the use of ``memory`` to enable caching becomes interesting when the
 fitting of a transformer is costly.
+"""
 
 # %%
-Illustration of ``Pipeline`` and ``GridSearchCV``
+# Illustration of ``Pipeline`` and ``GridSearchCV``
 ###############################################################################
 
-This section illustrates the use of a ``Pipeline`` with ``GridSearchCV``
-"""
-
 # Authors: Robert McGibbon, Joel Nothman, Guillaume Lemaitre
-
+#
+# License: BSD 3 clause
```

Comment:
Officially, I do not think we can add a license like this without author permission. To be safe, lets remove them:

```suggestion
```

Reply:
Since the full project is under this license, I assume that this is fine (but I am bad with licensing :))

Reply:
All the other `.py` files in this folder had the BSD license listed, so I added it for consistency:
https://github.com/scikit-learn/scikit-learn/tree/main/examples/compose

Reply:
I would like to have it for consistency. But I do not know if it is safe to do without the authors' permission.

If the whole project is under a license, does that mean we can add it to any file in the library without consent from the author? (I do not know the answer to this)